tests/common/devices.py

```diff
@@ -404,6 +404,26 @@ def get_critical_group_and_process_lists(self, container_name):
                 succeeded = False
                 break
 
+        # For PMon container, since different daemons are enabled in different platforms, we need find common processes
+        # which are not only in the critical_processes file and also are running in that platform.
+        if succeeded and container_name == "pmon":
+            common_groups = []
+            common_processes = []
```

Comment:
Suggest renaming from `common_*` to `expected_critical_*`

Reply:
Updated!